library/qs_source/unstable/src/StatePreparation.qs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export Microsoft.Quantum.Unstable.StatePreparation.PreparePureStateD, Microsoft.Quantum.Unstable.StatePreparation.ApproximatelyPreparePureStateCP;
```

Comment:
nit: even though it's just two exports, it is easier to see what they are if they are in different lines.

Reply:
I've updated these files entirely to not to re-exports.